app/overrides/add_favorite_products_tab.rb

```diff
@@ -1,6 +1,6 @@
 Deface::Override.new(
-  :virtual_path => 'spree/admin/shared/_product_sub_menu',
+  :virtual_path => 'spree/admin/shared/sub_menu/_product',
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.